meson.build

```diff
@@ -0,0 +1,81 @@
+project('libzseek', 'c',
+    default_options: ['c_std=c11', 'warning_level=3', 'werror=true'],
+    license: 'BSD-3-Clause',
+    version: '3.2.0')
```

Comment:
You can specify `meson_version: '>=0.56.0',` here as well, and too old versions of Meson will raise an informative error message stating that they are too old to even try building this project. Much better than cryptic invalid function errors. :)

I also pointed out two changes you can make to drop your minimum requirements to 0.49.0, which preserves compatibility with several popular and actively maintained linux distro releases, including Ubuntu 20.04 and Debian buster.

Reply:
Again, than you for the detailed and spot-on notice! Hadn't paid attention to the version requirement, but spent some time now: going ahead with `0.50.0` (`0.49.0` would require also changing the `include_directories:` argument to the `declare_dependency` function).

Testing on Debian buster made me realize that the zstd requirement for `>=1.4.9` is more of a blocker on older distros than meson after all, making the decision even easier :)

Reply:
```
mkdir subprojects
meson wrap install zstd
```

Builds zstd as a subproject, if it is not available on the system. The subproject is compatible with meson 0.47.0, but for users on <0.54 you'd need to add the `fallback: ['zstd', 'libzstd_dep']` kwarg since those old versions of Meson don't support the wrap file's `[provide]` mapping section.

Reply:
... scratch that, the wrapdb still has 1.4.5 because I keep on getting delayed from updating it due to issues getting it to build in a totally warning-free manner.

Reply:
No worries, thanks a lot, again!